CONTRIBUTING.md

```diff
@@ -90,19 +91,42 @@ Every non-trivial change to the code base should be accompanied by a
 relevant addition to or modification of the test suite.
 
 Please check that the full test suite (including your test additions
-or changes) passes successfully on your local machine before you
-open a pull request.
+or changes) passes successfully on your local machine **before you
```

Comment:
Do we really want to stress this? Why? Travis is there for a reason.

Reply:
Because it's preferable for the user to test on their machine rather than commit, submit, repeat until they pass the tests. Especially since we have a limited number of workers concurrently it would be better not to waste them doing testing that they can do on their machines.